components/StatusItem/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { Text, View, StyleSheet, Image } from 'react-native';
+import { Constants } from 'expo';
+import PropTypes from 'prop-types';
+
+
+class Icon extends Component {
+  render() {
+    if (this.props.status == 'uploaded') {
+    pic = {
+      uri: 'https://emojipedia-us.s3.amazonaws.com/thumbs/160/facebook/105/heavy-check-mark_2714.png'
```

Comment:
just upload the two pictures into the project in a photos folder and call that url